content/blog/2018/06/2018-06-15-anchore-image-scanning.adoc

```diff
@@ -0,0 +1,173 @@
+---
+:layout: post
+:title: "Securing your Jenkins CI/CD Container Pipeline with Anchore (in under 10 minutes)"
+:tags:
+- events
+- community
+- developer
+:author: michaelneale
+---
+
```

Comment:
Reference the original blogpost and author in the note?

Reply:
Hi @oleg-nenashev - this is new content (derived from an older post, but updated significantly), and is complete by itself,  so I do not think a reference is necessary.  I suggest the authors should include - Daniel Nurmi, Michael Neale, Andrew Cathrow

Reply:
@michaelneale did you have a chance to process this comment?

Reply:
Yes I did - I put a link below that